frontend/app/server.tsx

```diff
@@ -43,7 +43,14 @@ const renderAMPArticle = ({ body }: express.Request, res: express.Response) => {
     try {
         const CAPI = extractArticleMeta(body);
         const resp = AMPDocument({
-            body: <AMPArticle elements={CAPI.elements} pillar={CAPI.pillar} />,
+            body: (
+                <AMPArticle
+                    elements={CAPI.elements}
+                    pillar={CAPI.pillar}
+                    CAPI={CAPI}
```

Comment:
In a way, I'm keen for us to try and avoid just passing the full CAPI object (as it's too big/doesn't document dependencies well). But if we do, then we should remove the elements and pillars arguments. Ideal would be to pass in the subset needed for the header though.

Reply:
I've updated this with a more limited type